src/components/ContextualMenu/contextualmenu.js

```diff
@@ -0,0 +1,38 @@
+import React, {useState} from 'react';
+import './contextualmenu.css';
+import EditMovieModal from '../Modals/EditMovieModal/editmoviemodal';
+import DeleteMovieModal from '../Modals/DeleteMovieModal/deletemoviemodal';
+
+const ContextualMenu = ({ showMenu, xPos, yPos }) => {
+  
+  const [editShowModal, setEditShowModal] = useState(false);
+  const [deleteShowModal, setDeleteShowModal] = useState(false);
+  
+  const handleEditClose = () => setEditShowModal(false);
+  const handleEditShow = () => setEditShowModal(true);
+  
+  const handleDeleteClose = () => setDeleteShowModal(false);
+  const handleDeleteShow = () => setDeleteShowModal(true);
```

Comment:
I think you can use functional state updates for that:
https://reactjs.org/docs/hooks-reference.html#functional-updates

It would be like:
```
const toggleEdit = () => setEditShowModal(prevValue => !prevValue);
```

Reply:
fixed